gpfit/fit.py

```diff
@@ -1,13 +1,13 @@
 "Implements the all-important 'fit' function."
-from numpy import ones, exp, sqrt, mean, square, hstack
+from numpy import ones, exp, sqrt, mean, square, hstack, array
 from gpkit import NamedVariables, VectorVariable, Variable, NomialArray
 from .implicit_softmax_affine import implicit_softmax_affine
 from .softmax_affine import softmax_affine
 from .max_affine import max_affine
 from .levenberg_marquardt import levenberg_marquardt
 from .ba_init import ba_init
 from .print_fit import print_ISMA, print_SMA, print_MA
-
+import pandas as pd
```

Comment:
Maybe put this import in the `get_dataframe` function, so that a user only needs pandas installed if they want to call that?